docs/index.md

```diff
@@ -53,6 +53,7 @@ Alternatively, you can install Particula using conda:
 conda install -c conda-forge particula
 ```
 
+If you are new to Python or plan on going through the Examples, head to [Setup Particula](Examples/Setup_Particula/index.md) for more comprehensive installation instructions.
```

Comment:
Add an empty line before the page break --- . I guess right now it renders that as a title or something. https://uncscode.github.io/particula/pr-preview/pr-693/#if-you-are-new-to-python-or-plan-on-going-through-the-examples-head-to-setup-particula-for-more-comprehensive-installation-instructions

```suggestion
If you are new to Python or plan on going through the Examples, head to [Setup Particula](Examples/Setup_Particula/index.md) for more comprehensive installation instructions.

```

Reply:
Hah, whoops, thanks!